### ISSUE TITLE
Checked Icon button background is not highlighted in High contrast 

### ISSUE THREAD

**Issue by shaikshavalis** [OfficeDev/office-ui-fabric-react#5793]
### Bug Report 
 
- __Package version(s)__: 5.112.1004
- __Browser and OS versions__: Edge/Windows 10
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: Normal
 
Products/sites affected: vsts
 
#### Describe the issue:
 
Default State : 
![image](https://user-images.githubusercontent.com/9265381/43637871-28c66726-9734-11e8-8b73-f7942b8a591b.png)

Checked state : 
![image](https://user-images.githubusercontent.com/9265381/43637900-4ce1d8ac-9734-11e8-9cec-508fb9ab55e6.png)

High contrast Normal and Checked state: 
![image](https://user-images.githubusercontent.com/9265381/43637853-185d08a4-9734-11e8-80bf-48ede58a611d.png)

 
#### Actual behavior:
 
High contrast Normal and Checked state: 
![image](https://user-images.githubusercontent.com/9265381/43637853-185d08a4-9734-11e8-80bf-48ede58a611d.png)

 
#### Expected behavior:
 
Background should be highlighted for the button in high contrast to recognize whether it is checked or not.

**Comment by natalieethell** [comment]
Hey thanks for submitting this issue! It looks like we're actually not currently supporting a checked state in high contrast for *any* of our buttons (note that although "Mark as checked" is set, they all look the same, which is not the case when high contrast is off). 

![image](https://user-images.githubusercontent.com/14134000/43740431-0e2185ec-9980-11e8-96f1-9d18ce8549cf.png)

![image](https://user-images.githubusercontent.com/14134000/43740417-042333ec-9980-11e8-93fe-1a9fffefab34.png)

@betrue-final-final what are the expectations for how a checked button should look in HC?

**Comment by philipzloh** [comment]
Pressed button states are not a thing in High Contrast mode. This doesn't need Design, this is impossible ;)

![image](https://user-images.githubusercontent.com/836760/43790188-b623369a-9a26-11e8-8174-2a9c8eecbf15.png)

**Comment by natalieethell** [comment]
Thanks @philipzloh ! Given that visual pressed button states are not present in HC, we'll keep what we currently have in Fabric. 

A visual effect would require some type of glyph change on the page (e.g. a check in a checkbox). We do have an `aria-pressed` attribute on all of our buttons with `toggle` prop set to true that can keep track of the pressed state (thanks @JasonGore for [this PR](https://github.com/OfficeDev/office-ui-fabric-react/pull/5825)).

I'll go ahead and close out this issue, but let us know if you have any further questions!
